hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/ViewFileSystem.java

```diff
@@ -917,13 +917,9 @@ public void removeXAttr(Path path, String name) throws IOException {
   }
 
   @Override
-  public void setVerifyChecksum(final boolean verifyChecksum) { 
-    List<InodeTree.MountPoint<FileSystem>> mountPoints = 
-        fsState.getMountPoints();
-    Map<String, FileSystem> fsMap = initializeMountedFileSystems(mountPoints);
-    for (InodeTree.MountPoint<FileSystem> mount : mountPoints) {
-      fsMap.get(mount.src).setVerifyChecksum(verifyChecksum);
-    }
+  public void setVerifyChecksum(final boolean verifyChecksum) {
```

Comment:
Looks like the default impl. in FileSystem is already no-op. So the value of overriding is only in the comment. May be we should just remove this methods and add a comment to the jira explaining the change.

Reply:
We still want to keep overriding the two methods in ViewFileSystem (same as ViewFs) so that we explicitly say these are no-ops and doesn't depend on the FileSystem implementation (if anything changes in future)